Makefile

```
@@ -45,69 +45,58 @@
 # ==============================================================================
 
 .PHONY: setup
-setup: ## Install dependencies and create .env files from .env.example
+setup: ## Install dependencies and create .env file if it doesn't exist
```

checkmake (0.2.2)

[warning] Target body for "setup" exceeds allowed length of 5 (7).

(maxbodylength)